### ISSUE TITLE
Login issue with new installation

### ISSUE THREAD

**Issue by arcotbha** [ushahidi/SwiftRiver#341]
Hi everybody,

I've been having login issues with new Swiftriver installation. When I try to login with the default username and password i.e., admin and password, it always renders a failure notice saying "Failure The username and password you entered is incorrect". Am I giving a wrong password or is it a problem with my installation? 

note: I installed latest of integration-dev. Also in the login page it always asks me for an "Email address and password" instead of "username and password", are they one and the same?

![loginerror](https://f.cloud.github.com/assets/3100096/398702/c59b8840-a85d-11e2-9147-b796b04fc2b4.png)

Regards,
Preethi

**Comment by arcotbha** [comment]
Any idea on this friends?

**Comment by stonemirror** [comment]
I'm stuck on this same issue. I guess I'll have to dig into the code. Meh.

**Comment by arcotbha** [comment]
Yeah @stonemirror... I wasn't able to figure out why and I quit working on it long back...  but still would appreciate if somebody could help....

**Comment by lorenzotlc** [comment]
Same problem. Installed following the wiki from github at the end of August.
Any help?

Is there any swiftriver / swiftriver-api installation working?
